cookies/operations.py

```diff
@@ -12,49 +12,15 @@
 import networkx as nx
 
 import goat
-import unittest, mock, json, os, sys
+import os
 
 from cookies.exceptions import *
 
 logger = settings.LOGGER
 
 os.environ.setdefault('GOAT_WAIT_INTERVAL', '0.001')
-
-goat.GOAT_APP_TOKEN = 'd22bbda9b5b507dc6cd032d80d6a3d299fda10fe'
-goat.GOAT = 'http://127.0.0.1:8000'
-
-class MockResponse(object):
-    def __init__(self, content, status_code):
-        self._status_code = status_code
-        self.content = content
-
-    def json(self):
-        return json.loads(self.content)
-
-    @property
-    def status_code(self):
-        return self._status_code
-
-
-class MockSearchResponse(MockResponse):
-    url = 'http://mock/url/'
-
-    def __init__(self, parent, pending_content, success_content, max_calls=3,):
-        self.max_calls = 3
-        self.parent = parent
-        self.pending_content = pending_content
-        self.success_content = success_content
-
-    def json(self):
-        if self.parent.call_count < self.max_calls:
-            return json.loads(self.pending_content)
-        return json.loads(self.success_content)
-
-    @property
-    def status_code(self):
-        if self.parent.call_count < self.max_calls:
-            return 202
-        return 200
+goat.GOAT_APP_TOKEN = os.environ.get('GOAT_APP_TOKEN','')
+goat.GOAT = os.environ.get('GOAT_URI','')
```

Comment:
This is good. I'd go one step further, and load the environment variable in the settings module, and then set the app token and endpoint (here) from a settings variable.